src/main/java/com/minecolonies/commands/DeleteColonyCommand.java

```diff
@@ -0,0 +1,129 @@
+package com.minecolonies.commands;
+
+import com.minecolonies.colony.Colony;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.Colony'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)